### DISCUSSION TITLE
Not able to get my student developer pack!

### DISCUSSION THREAD

**Discussion by shrey802** [community/community#60352] [GitHub Education]
### Hi everyone,

I've reapplied to the student developer pack and my request has been approved. But it says that I have to wait for the benefits. I don't have to reapply but there's a waiting time for the benefits. 
![githubproof (1)](https://github.com/community/community/assets/55155180/3002c031-d90f-420a-b977-f988f24d479a)
![githubproof (2)](https://github.com/community/community/assets/55155180/4f221e6e-1e7d-40e0-a88d-bc04e11c12c3)

What should I do? Should I reapply or wait? It's already been 4 days

**Comment by GhostOf0days** [comment]
Please create a ticket with [Github Support](https://support.github.com/) by clicking 'My Tickets' in the top right.

**Reply by zxcvb51226** [reply]
> Please create a ticket with [Github Support](https://support.github.com/) by clicking 'My Tickets' in the top right.

**Reply by zxcvb51226** [reply]
> > Please create a ticket with [Github Support](https://support.github.com/) by clicking 'My Tickets' in the top right.
> 
>

**Comment by samus-aran** [comment]
Hey @shrey802  👋

We've updated our application and re-verification process to ensure that GitHub and our partners can offer exceptional, free benefits to students worldwide during their academic journey.

With this update, when you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive within 72 hours of this verification. However, during busy periods like the start of a new semester, it may take a bit longer.

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!

I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
